changelog/db.changelog-1.0.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.0.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <createTable tableName="examples">
+      <column name="exampleId" type="INTEGER" autoIncrement="true" >
+        <constraints nullable="false" primaryKey="true" primaryKeyName="exampleId_pkey" />
+      </column>
+      <column name="textField" type="VARCHAR(50)" />
+    </createTable>
+  </changeSet>
+</databaseChangeLog>
```

Comment:
Example of how to create a new table using Liquibase.  This one correlates to the `example.js` model in `app/data/models` folder.

Feel free to delete before merging.